packages/wallet/src/services/HandlesTracker.ts

```diff
@@ -83,5 +128,19 @@ export const createHandlesTracker = ({ tip$, assetInfo$, handlePolicyIds, logger
       );
     }),
     distinctUntilChanged((a, b) => sameArrayItems(a, b, handleInfoEquals)),
+    map((handles) =>
+      from(handles).pipe(
+        concatMap((handle) =>
+          of(hydrateHandle(handleProvider, logger)(handle)).pipe(
+            delay(RESOLVE_HANDLE_DELAY),
+            mergeMap((x) => x)
+          )
+        ),
+        take(handles.length),
```

Comment:
I think this is redundant because `from(handles)` will complete by itself after emitting all handles
```suggestion
```

Reply:
Fixed here: https://github.com/input-output-hk/cardano-js-sdk/commit/2ecb53301b9c37c6727088f9fe3ac58c4d2592db.